setup.cfg

```diff
@@ -0,0 +1,26 @@
+[bumpversion]
```

Comment:
take a not from hangar's `bumpversion` config linked [here](https://github.com/tensorwerk/hangar-py/blob/master/.bumpversion.cfg) and just copy/paste all but the specific replace portions. This let's you bump to prerelease (alpha/beta/rc) candidates as well as just standard versions.

Reply:
Yep, in my TODO already for next minor. Thanks